manifests/e/e2eSoft/e2eSoftiVCam/6.2.2.0/e2eSoft.e2eSoftiVCam.locale.en-US.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: e2eSoft.e2eSoftiVCam
+PackageVersion: 6.2.2.0
+PackageLocale: en-US
+Publisher: e2eSoft
+PackageName: e2eSoft iVCam
+License: (c) e2eSoft
+ShortDescription: e2eSoft iVCam Setup

```

Let's add a description.

```suggestion
ShortDescription: iVCam turns your Phone/Pad into an HD webcam for Windows PC.
```